docs/user/07-70-function-specification.md

```diff
@@ -0,0 +1,261 @@
+
+This example shows how to set such a custom response in Node.js for the HTTP status code `400`:
+
+```js
+module.exports = {    
+    main: function (event, context) {
+        if (event.extensions.request.query.id === undefined) {
+            res = event.extensions.response;
+            res.status(400)
+            return
+        }
+        return "42"
+    }
+}
+```
+
+</details>
+<details>
+<summary label="Python">
+Python
+</summary>
+
+By default, a failing Function simply throws an error to tell the Event Service to reinject the event at a later point. Such an HTTP-based Function returns the HTTP status code `500`. In case you manage to invoke a Function successfully, the system returns the default HTTP status code `200`.
+
+Apart from these two default codes, you can define custom responses in all Python runtimes using the **HTTPResponse** object available in Bottle.
+
+This object will need to be instantiated and can be customized. For more information, read [Bottle API documentation](https://bottlepy.org/docs/dev/api.html#the-response-object).
+
+The following example shows how to set such a custom response in Python for the HTTP status code `400`:
+
+```python
+from bottle import HTTPResponse
+
+SUPPORTED_CONTENT_TYPES = ['application/json']
+
+def main(event, context):
+    request = event['extensions']['request']
+
+    response_content_type = 'application/json'
+    headers = {
+        'Content-Type': response_content_type
+    }
+
+    status = 202
+    response_payload = {'success': 'Message accepted.'}
+
+    if request.headers.get('Content-Type') not in SUPPORTED_CONTENT_TYPES:
+        status = 400
+        response_payload = json.dumps({'error': 'Invalid Content-Type.'})
+
+    return HTTPResponse(body=response_payload, status=status, headers=headers)
+```
+
+</details>
+</div>
+
+## /metrics endpoint  
+
+You can use the `/metrics` endpoint to return the Function metrics. All the information is gathered using Prometheus and can be displayed using the Grafana dashboard (see [Kyma observability](https://kyma-project.io/docs/kyma/latest/02-get-started/05-observability/) for more information on how to use Grafana dashboard in Kyma). As this endpoint is provided by Kubeless, it cannot be customized.  
+For more information, see [Kubeless monitoring](https://github.com/vmware-archive/kubeless/blob/master/docs/monitoring.md) and [Kubeless runtime variants](https://github.com/vmware-archive/kubeless/blob/master/docs/runtimes.md) pages.
+
+> **NOTE:** Prometheus and Grafana are [deprecated](https://kyma-project.io/blog/2022/12/9/monitoring-deprecation) and are planned to be removed. If you want to install a custom stack, take a look at [Install a custom kube-prometheus-stack in Kyma](https://github.com/kyma-project/examples/tree/main/prometheus).
+
```

Comment:
```suggestion
```

Unfortunately the endpoint is no longer available (since we removed kubeless from nodejs)